### DISCUSSION TITLE
ApiError on video upload with 1.109.2

### DISCUSSION THREAD

**Discussion by Str1atum** [immich-app/immich#11242] [Q&A] [answered]
Since 1.109.2 I fail uploading videos to immich server via web interface. Even with only around 150 mb it doesn't work any more. Photos do upload without issues. I use traefik 3.1 but don't have issues with other services, no Cloudflare.

**Comment by akoyaxd** [comment]
Same issue here. Looks like uploads get cancelled after exactly one minute. Did traefik added recently a max send time? Or did maybe immich add one?

**Reply by akoyaxd** [reply]
So I can not find any timeout at traefik docs that defaults to 60s

**Reply by akoyaxd** [reply]
@bo0tzz is it possible, that the immich docker image has something like max request time on its own?
And maybe traefik is not buffering anything by default so it has longer request times.

**Reply by akoyaxd** [reply]
/api/assets gives me back a 502 from traefik. But I can not see the request on the immich server container.

**Reply by akoyaxd** [reply]
I see it in the traefik logs though..

**Reply by akoyaxd** [reply]
After adding the buffering middleware to immich get 500 errors in traefik.
      - "traefik.http.middlewares.limit.buffering.maxRequestBodyBytes=100000000000"
      
   Still after exactly 60 seconds

**Reply by RogerSik** [reply, marked as answer]
Yes see here: https://github.com/immich-app/immich/discussions/8872

**Reply by akoyaxd** [reply]
Uff, sorry for not finding this! Thanks for the hint.
